indico/modules/rb_new/util.py

```diff
@@ -49,3 +60,23 @@ def search_for_rooms(filters, only_available=False):
     else:
         rooms = query.all()
     return rooms
+
+
+def get_buildings(location=None):
+    buildings = defaultdict(dict)
+    rooms = location.rooms if location else Room.find_all()
```

Comment:
`Room.query.filter_by(is_active=True).all()` (let's try to avoid adding more `Foo.find*()` calls, we kind of deprecated them quite some time ago_

Reply:
Good to know ;)